dir.props

```diff
@@ -67,6 +67,9 @@
     <PackagesBinDir>$(__PackagesBinDir)</PackagesBinDir>
     <PackagesBinDir Condition="'$(__PackagesBinDir)'==''">$(BinDir).nuget\</PackagesBinDir>
 
+    <PackageInstallPath>$(__PackageInstallPath)</PackageInstallPath>
```

Comment:
In the packaging.props would be the place I think to set it. It needs to be set in all the pkgprojs essentially.